src/crimson/admin/admin_socket.cc

```diff
@@ -415,6 +415,104 @@ class InjectArgsHook final : public AdminSocketHook {
   }
 };
 
+/**
```

Comment:
I'm assuming all this code does is actually code shuffling.

Reply:
yes, what the commit of "crimson/admin: move config hooks into admin_socket.cc" does is just moving the code around.